src/tls/Makefile-tls.am

```diff
@@ -95,9 +95,11 @@ test_tls_server_SOURCES = \
 	$(NULL)
 
 test_tls_server_CFLAGS = $(TEST_CFLAGS)
```

Comment:
since we're being picky: strictly speaking, pthreads-using programs should also have `-pthread` in the `CFLAGS`

can you add it as well for `cockpit-tls` itself?